tokio-util/src/io/mod.rs

```diff
@@ -1,16 +1,22 @@
 //! Helpers for IO related tasks.
 //!
-//! These types are often used in combination with hyper or reqwest, as they
+//! The stream types are often used in combination with hyper or reqwest, as they
 //! allow converting between a hyper [`Body`] and [`AsyncRead`].
 //!
+//! The [`ReadBridge`] and [`WriteBridge`] convert from the world of async I/O
```

Comment:
I think this ought to be updated now that there's a single type rather than two?